system/settings/settings.xml

```diff
@@ -661,6 +661,18 @@
           <default>0</default>
           <control type="slider" format="percentage" range="0,100" />
         </setting>
+        <setting id="subtitles.linespacing" type="number" label="39202" help="39203">
+          <level>3</level>
+          <default>8</default>
+          <constraints>
+            <minimum>0</minimum>
+            <step>0.5</step>
+            <maximum>100</maximum>
+          </constraints>
+          <control type="slider" format="percentage">
+            <format>{:.1f} %</format>
+          </control>
+        </setting>
```

Comment:
why we should have a 0.1 precision with 0.5 steps?
i have done a test by using it as integer and at least on a 1080p display i dont see so much difference

Reply:
It felt like with a 1.5 scale factor the adjacent values could be too far apart, e.g 10*1.5=15, 11*1.5=16.5 (17?), 12*1.5=18.

Reply:
you can never be sure of moving by 1px, there is no always a fixed value of 1.5, and also can be scaled

from my end and based on what i see on the display by testing 720p and 1080p and overrides,
i don't see the need to have this precision, even if it moves 2 px at time it's not a very significant thing

i would like to have a feedback on this from someonelse of team, maybe @enen92?

Reply:
I don't have a strong opinion on this, can switch it to integers if that's the only thing that stops this PR from being merged

Reply:
Done, switched to integer and rebased on master